src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,22 @@
+const ADD_ROCKET = 'spaceTravelersHub/rockets/ADD_ROCKET';
+
+const initialState = {
+  rockets: [],
+  missions: [],
+};
+
+export const addRocket = (payload) => ({
+  type: ADD_ROCKET,
+  payload,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_ROCKET:
+      return { ...state, rockets: [...state.rockets, action.payload] };
+    default:
+      return state;
+  }
+};
```

Comment:
I was testing some things and found that redux is particularly intelligent when it comes to organizing data. Let me explain. 
When we combine the reducers using 
```javascript
    ... = combineReducers({
    missionsReducer,
    rocketsReducer,
})
```
The store ```state``` becomes something like this.

```javascript
    state = {
    missionsReducer: // local reducer state,
    rocketsReducer: // local reducer state,
}
```
This means we don't need to keep track of each others data. You can safely use an array as the state and keep your rocket's data inside it, without interfering with the mission's data I'm currently working on. 

I think it would be better to change it because this way the code will look more organized and accessing the data will be easier when we'll work on it. Please let me know if you have any questions or suggestions about this.